doc/output-validation.ipynb

```diff
@@ -0,0 +1,830 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "04e9acd5-d690-465b-9f5d-c3ab854e6d0e",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "<img align=\"left\" src = https://linea.org.br/wp-content/themes/LIneA/imagens/logo-header.jpg width=130 style=\"padding: 20px\"> \n",
+    "\n",
+    "# PZ-Validation output with using RAIL\n",
+    "\n",
+    "   **Contact**: Heloisa da S. Mengisztki, Julia Gschwend<br>\n",
+    "   **Last verified run**: 2024-june <br><br><br>\n",
+    "\n",
+    "QA created to validate the quality of outputs produced by LIneA PZ-Compute pipeline, documentedin the Stage III - Photo-z computing in [S4.4 - PZ Tables as Federated Datasets](https://linea-it.github.io/pz-lsst-inkind-doc/s4_4/). This notebook was based on the notebooks present in the rail documentation \"Goldenspike\", \"Evaluation_by_type\" and \"Evaluation_Demo\", which can be found in the Rail repository available at [Rail-Github](https://github.com/LSSTDESC/rail). Reference notebooks authors: RAIL team.\n",
+    "\n",
+    "### Table of Contents\n",
+    "1. [Imports](#Imports)\n",
+    "2. [Reading the files](#Reading-the-files)\n",
+    "3. [Metrics Validation](#Metrics-Validation )\n",
+    "   - Sumarry Statistics\n",
+    "   - CDF based Metrics\n",
+    "4. [Plots](#Plots)\n",
+    "   - Redshift stacked distribution of the pdfs\n",
+    "   - Zphot vs. Ztrue\n",
+    "   - QQ Plot\n",
+    "   - KS Plot\n",
+    "5. [Summarized Metrics](#Summarized-Metrics)\n",
+    "\n",
+    "-----\n",
+    "\n",
+    "### Observations and Pre requisites\n",
```

Comment:
Later on, I suggest to add some lines using pzserver, so it might be added as a pre requisite too.

Reply:
added the suggestion